example/src/pages/Dashboard/Transactions/index.tsx

```diff
@@ -22,14 +29,15 @@ const Transactions = () => {
       timeout: 3000,
       contractAddress
     }).then(({ data, success }) => {
+      refreshAccount();
       setState({
         transactions: data,
         transactionsFetched: success
       });
     });
   };
 
-  React.useEffect(fetchData, []);
+  React.useEffect(fetchData, [transactionsStatuses.successful]);
```

Comment:
Please make sure that fetchData runs only if successful is true, not on false, that would be an extra request

Reply:
I tested this but we've got an issue with the transactions not loading anymore on the initial load because the boolean return `false` when you first load the page.